budget_invoice/model/budget_line.py

```diff
@@ -46,10 +46,34 @@ def invoice_create(self, cr, uid, ids, product, partner, context=None):
         invoice_id = invoice_obj.create(cr, uid, invoice_data, context)
 
         for budget_line in self.browse(cr, uid, ids, context):
+            product_onchange_result = invoice_line_obj.product_id_change(
+                cr, uid, [],
+                product.id,
+                uom_id=False,
+                qty=0,
+                name=budget_line.name or u'/',
+                type='in_invoice',
+                partner_id=partner.id,
+                fposition_id=False,
+                price_unit=budget_line.amount,
+                currency_id=False,
+                context=context,
+                company_id=(budget_line.analytic_account_id and
+                            budget_line.analytic_account_id.company_id.id or
+                            None),
+            )
             invoice_line_data = {
```

Comment:
just v7 to v8 migration for modules to be available.
maybe in a next PR specific for v8 api ?